cpp/density/RDF.h

```diff
@@ -17,7 +17,24 @@ class RDF : public locality::BondHistogramCompute
 {
 public:
     //! Constructor
-    RDF(unsigned int bins, float r_max, float r_min = 0, bool normalize = false);
+    RDF(unsigned int bins, float r_max, float r_min = 0, bool normalize = false){
```

Comment:
The constructor is implemented in `RDF.cc`. The header file should only contain the declaration of the constructor and its arguments. Please revert these changes in `RDF.h` and add the block for `if (r_min < 0)` to the `RDF.cc` file instead.